src/test/scala/com/github/swagger/akka/samples/DogsHttpService.scala

```diff
@@ -17,22 +17,27 @@ package com.github.swagger.akka.samples
 
 import javax.ws.rs.Path
 
-import scala.reflect.runtime.universe._
-
-import com.github.swagger.akka._
-import com.github.swagger.akka.model.{License, Contact, Info}
-
 import akka.actor.{ActorRefFactory, ActorSystem}
-import akka.http.scaladsl.server.Directives
+import akka.http.scaladsl.model.ContentTypes.`application/json`
+import akka.http.scaladsl.model.HttpMethods.OPTIONS
+import akka.http.scaladsl.model.StatusCodes.OK
+import akka.http.scaladsl.model.headers.`Access-Control-Allow-Methods`
+import akka.http.scaladsl.model.{HttpEntity, HttpResponse}
+import akka.http.scaladsl.server.{Directives, Route}
+import com.github.swagger.akka._
+import com.github.swagger.akka.model.{Contact, Info, License}
 import io.swagger.annotations._
 
+import scala.reflect.runtime.universe._
```

Comment:
Could you order these imports in the old way? java, scala, org, com, everything else